components/currencyfield.js

```diff
@@ -0,0 +1,26 @@
+const TextField = require('./textfield')
+const toCurrency = require('../lib/tocurrency')
+
+class CurrencyField extends TextField {
```

Comment:
As a `TextField` this inherits the maximum length property - is that something that is valid for a currency field? If it is (it does help prevent "sat on keyboard"-type errors), should we set it automatically rather than asking the developer for it?